code/modules/mob/living/carbon/alien/adult/queen.dm

```diff
@@ -181,3 +181,15 @@
 	if(!silent)
 		to_chat(user, span_noticealien("You discard [src]."))
 	return ..()
+
+/mob/living/carbon/alien/adult/royal/queen/attackby(obj/item/O, mob/user, params)
+	if(istype(O, /obj/item/clothing/under/costume/maid) || istype(O, /obj/item/clothing/under/rank/civilian/janitor/maid))
```

Comment:
should this be a typecache

Reply:
idk it works.

Reply:
it would make adding new cases less verbose, and overall just nicer

Reply:
actually no scratch that bad idea I take it back